lib/usd/translators/shading/mtlxMaterialXSurfaceShaderWriter.cpp

```diff
@@ -26,6 +26,9 @@
 #include <pxr/usd/usdShade/material.h>
 #include <pxr/usd/usdShade/nodeGraph.h>
 #include <pxr/usd/usdUI/nodeGraphNodeAPI.h>
+#if PXR_VERSION >= 2505
+#include <pxr/usd/usdMtlx/tokens.h>
+#endif
```

Comment:
That is on the USD side. In 25.05 we will need this include to get the usdMtlx tokens.